templates/partials/answers/mobilenumber.html

```diff
@@ -1,23 +1,23 @@
 {% from "components/input/_macro.njk" import onsInput %}
 
 {% set input = form.fields[answer.id] if input is undefined else input %}
-
 {% set config = {
-  "id": answer.id,
-  "name": input.name,
-  "type": "tel",
-  "autocomplete": "tel",
-  "width": "15",
-  "value": input._value() | e,
-  "label": {
-    "id": answer.id ~ "-label",
-    "text": answer.label,
-    "description": answer.description
-  },
-  "attributes": {
-    "data-qa": "input-text"
-  },
-  "error": error
+    "id": answer.id,
+    "name": input.name,
+    "type": "tel",
+    "autocomplete": "tel",
+    "width": "15",
+    "value": input._value() | e,
+    "label": {
+        "id": answer.id ~ "-label",
+        "text": answer.label,
+        "description": answer.description
+    },
+    "attributes": {
+        "data-qa": "input-text"
+    },
+    "error": error
 } %}
-
+{# djlint:off #}
 {{ onsInput(config) }}
+{# djlint:on #}
```

Comment:
Do we need to turn it off and on around this one line?

Reply:
Yep, otherwise the config is generates with some djlint specific string.